### PR TITLE
Add euserv provider

### PR DESCRIPTION
**Pull request by mschoettle**
This PR adds support for EUserv (https://support.euserv.com/api-doc/)

### CODE REVIEW

**Comment by mschoettle** [comment]
I addressed all the issues from the checks. The remaining exception in python-3.8-cover is `UnicodeDecodeError: 'utf-8' codec can't decode byte 0x9b in position 27: invalid start byte`. Is this due to a broken build script or is there anything in data that causes this?

**Comment by adferrand** [comment]
Hello @mschoettle!

Sorry for the delay to respond. I will study the situation tomorrow.

Still, I am very surprise that the error is occurring on only one version of Python, and only one system. Could you relaunch a second time the CI, to check if it was not a spurious failure?

**Comment by mschoettle** [comment]
@adferrand Good idea. I relaunched with a force push (couldn't find another way) and the error is gone now.

File: lexicon/providers/euserv.py [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+            data = {'dns_record_id': record_id}
+
+            response = response = self._get('kc2_domain_dns_remove', data)
+            LOGGER.debug('delete_record: %s', response)
+
+        return True
+
+    # Helpers
+    # url param used as subaction
+    def _request(self, action='GET', url='/', data=None, query_params=None):
```
Comment (adferrand) [inline lexicon/providers/euserv.py]: I would prefer we do not mutate the contract around `_request`. Indeed this function is part of the `BaseProvider`, and is overridden by all providers, giving a global consistency.

Technically, you are always using `url=/`, and so you never need to modify it. Could you pass the subaction directly in the `query_params` instead?

Comment (mschoettle) [reply lexicon/providers/euserv.py]: Good point, that makes sense. I changed it.

**Comment by mschoettle** [comment]
> LGTM!

Thanks! How will it be merged in? Do I have to close this PR?

**Comment by adferrand** [comment]
I will merge it through our release tool, but to make a new release, by the end of today!

**Pull request merged**